jest.config.js

```diff
@@ -12,10 +12,22 @@ module.exports = {
   runner: supportsESMAndJestLightRunner ? "jest-light-runner" : "jest-runner",
 
   collectCoverageFrom: [
-    "packages/*/src/**/*.{js,mjs,ts}",
-    "codemods/*/src/**/*.{js,mjs,ts}",
-    "eslint/*/src/**/*.{js,mjs,ts}",
+    "packages/*/src/**/*.{js,cjs,mjs,ts}",
+    "codemods/*/src/**/*.{js,cjs,mjs,ts}",
+    "eslint/*/src/**/*.{js,cjs,mjs,ts}",
   ],
+  // coveragePathIgnorePatterns Doesn't work
```

Comment:
This is probably a jest bug, I can't find a way to make it work, even using `collectCoverageFrom:["!packages/babel-core/src/vendor/**"]`

Reply:
Maybe we need something like this to handle windows paths?
```js
coveragePathIgnorePatterns: [
  /packages[\\/]babel-core[\\/]src[\\/]vendor[\\/].*/
]
```

Reply:
No, jest only supports regex strings. Even if I use ".*", it still doesn't work, so I guess it's a bug, there's been a similar issue in the jest repo for several years.